python/podio_gen/cpp_generator.py

```diff
@@ -91,14 +91,16 @@ def pre_process(self):
 
         return {}
 
-    def post_process(self, _):
+    def post_process(self, datamodel):
         """Do the cpp specific post processing"""
         self._write_edm_def_file()
 
         if "ROOT" in self.io_handlers:
             self._prepare_iorules()
             self._create_selection_xml()
 
+        if datamodel["links"]:
+            self._write_links_registration_file(datamodel["links"])
```

Comment:
```suggestion
        if theLinks := datamodel.get("links"):
            self._write_links_registration_file(theLinks)
```

Reply:
Done. Plus bumped minimal python requirements to 3.8 in README